meinberlin/apps/budgeting/assets/ControlBar.jsx

```diff
@@ -94,6 +94,7 @@ export const ControlBar = props => {
                   : 'btn btn--light'
               }
               aria-label={translated.toggleFilters}
+              data-testid="id_toggle-filters"
```

Comment:
i don't like to add this explicitly, but not sure how to do it better?

Reply:
We should be able to use `findByLabelText` to find the element by aria-label if we change the mock of `gettext` a little. I will push a PR for that and then it should work if you use it here

Reply:
pushed https://github.com/liqd/a4-meinberlin/pull/4541